src/diploma.c

```diff
@@ -0,0 +1,254 @@
+#include "constants/fanfares.h"
+#include "gba/macro.h"
+#include "global.h"
+#include "gpu_regs.h"
+#include "window.h"
+#include "main.h"
+#include "scanline_effect.h"
+#include "task.h"
+#include "unk_text_util.h"
+#include "gpu_regs.h"
+#include "text.h"
+#include "bg.h"
+#include "sound.h"
+#include "malloc.h"
+#include "sprite.h"
+#include "palette.h"
+#include "new_menu_helpers.h"
+#include "menu.h"
+#include "diploma.h"
+
+void sub_80568FC(void);
```

Comment:
You may want to declare these functions in corresponding header files and include it. e.g. overworld.h

Reply:
@gwimm please address this kthx